### ISSUE TITLE
Possible differences between HookTest TEI DTD and oXygen default TEI validation

### ISSUE THREAD

**Issue by nevenjovanovic** [Capitains/HookTest#52]
When running HookTest (compiled from Github head, locally, from the command line, in a python virtualenv) on the repository [croala-pelagios](https://github.com/nevenjovanovic/croala-pelagios) (`hooktest -s 'tei' -v -c croala-pelagios`), some TEI documents don't validate, though they validate OK in default oXygen 18.0 TEI validation (TEI P5 3.0.0).
An example of test failure report:

```
>>>> Testing /bunic02/croala1761880/bunic02.croala1761880.croala-lat1.xml
>>>>> File parsing passed
>>>>>> Parsed
>>>>> URN informations passed
>>>>> Naming conventions passed
>>>>> RefsDecl parsing passed
>>>>>> 7 citation's level found
>>>>> Passage level parsing passed
>>>>>> 3 found
>>>>>> 7 found
>>>>>> 1021 found
>>>>>> 76 found
>>>>>> 174 found
>>>>>> 17 found
>>>>>> 24 found
>>>>> Unique nodes found by XPath passed
>>>>> Available in inventory passed
>>>>> TEI DTD Validation failed
>>>>>> error: element "app" not allowed here [In (L195 C21); (L209 C21); (L1571 C20)]
>>>>>> error: element "div" not allowed here [In (L197 C57); (L212 C63); (L1573 C57)]

```

**Comment by PonteIneptique** [comment]
Could you update https://github.com/Capitains/HookTest/blob/master/HookTest/resources/tei.rng through a PR ? :)

**Comment by PonteIneptique** [comment]
Fixed by #53
